src/tailwind/plugins/theme/gc-colors.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * SAA colors
+ */
+module.exports = function () {
+  return {
+    'gc-electric-blue': {
+      DEFAULT: '#505EEC',
+      light: '#6B77F5',
+    },
+  };
+};
```

Comment:
These are just temp utilities to test out the TW installation. I should've updated the comment at the top 🤦🏼‍♀️